es_ES/technical-overview.md

```diff
@@ -0,0 +1,156 @@
+---
+title: Resumen Técnico
+layout: page
+permalink: /technical-overview/
+---
+
+## ¿Qué es Flutter?
+
+Flutter es un SDK de aplicaciones móviles para la creación de aplicaciones de alto rendimiento y alta fidelidad para iOS y Android, a partir de un único codebase.
+
+El objetivo es permitir a los desarrolladores que proporcionen aplicaciones de alto rendimiento que se adapten de forma natural a diferentes plataformas. Aceptamos las diferencias en los comportamientos de scrolling, tipografía, iconos, y más.
```

Comment:
que lancen aplicaciones de alto rendimiento

Reply:
Flutter is a mobile app SDK for building high-performance, high-fidelity, apps for iOS and Android, from a single codebase.
No tiene lanzar sino construir o crear